contrib/yambar/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "yambar"
+pkgver = "1.11.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["bison", "flex", "meson", "pkgconf", "scdoc"]
+makedepends = [
+    "alsa-lib-devel",
+    "fcft-devel",
+    "json-c-devel",
+    "libinput-devel",
+    "libmpdclient-devel",
+    "libxcb-devel",
+    "libyaml-devel",
```

Comment:
you also want libpulse-devel in this list probably

also add explicit -Dthing=enabled in configure_args for these features